Src/Plugins/Security/Security.SharedStrings.txt

```diff
@@ -28,6 +28,7 @@
   $SEC101/003.GoogleApiKey=\b(?<refine>AIza(?i)[0-9a-z-_]{35})([^0-9a-z-_]|$)
   $SEC101/004.FacebookAppCredentials=(?si)facebook.{1,50}\b(?<id>[0-9]{15}).{1,500}?\b(?<secret>[0-9a-f]{32})
   $SEC101/005.SlackToken=\b(?<refine>xox(?<type>p|b|a|o|r|s)-(?i)[0-9a-z\-]+)
+  $SEC101/006.GitHubPat_2021_04_05=gh(p|o|u|s|r)_(?<secret>[\w]{30})(?<checksum>[\w]{6})
```

Comment:
>2021_04_05 [](start = 24, length = 10)

where is this date coming from? is it an official GH date for this change? is that how GitHub versions this PAT format revision?

Reply:
See my suggestion, use GitHubPatLegacy and GitHubPatCurrent

---
In reply to: [608156601](https://github.com/microsoft/sarif-pattern-matcher/pull/366#discussion_r608156601) [](ancestors = 608156601)

Reply:
Done.

---
In reply to: [608158992](https://github.com/microsoft/sarif-pattern-matcher/pull/366#discussion_r608158992) [](ancestors = 608158992,608156601)